packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -303,77 +242,6 @@ QUnit.test('`if` helper with inline form: can use falsy param as binding', funct
   equal(view.$().text(), 'no');
 });
 
-QUnit.test('`if` helper with inline form: raises when using more than three arguments', function() {
-  view = EmberView.create({
-    conditional: true,
-    template: compile('{{if one two three four}}')
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /The inline form of the `if` and `unless` helpers expect two or three arguments/);
-});
-
-QUnit.test('`if` helper with inline form: raises when using less than two arguments', function() {
-  view = EmberView.create({
-    conditional: true,
-    template: compile('{{if one}}')
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /The inline form of the `if` and `unless` helpers expect two or three arguments/);
-});
-
-QUnit.test('`if` helper with inline form: works when used in a sub expression', function() {
-  view = EmberView.create({
-    conditional: true,
-    innerConditional: true,
-    template: compile('{{if view.conditional (if view.innerConditional "truthy" )}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'truthy');
-});
-
-QUnit.test('`if` helper with inline form: updates if condition changes in a sub expression', function() {
-  view = EmberView.create({
-    conditional: true,
-    innerConditional: true,
-    template: compile('{{if view.conditional (if view.innerConditional "innerTruthy" "innerFalsy")}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'innerTruthy');
-
-  run(function() {
-    view.set('innerConditional', false);
-  });
-
-  equal(view.$().text(), 'innerFalsy');
-});
-
-QUnit.test('`if` helper with inline form: can use truthy param as binding in a sub expression', function() {
```

Comment:
replaced with module: `Helpers test: {{if}} used with another helper` and test: `it renders the corresponding block based on the conditional`